Frame.cc

```diff
@@ -635,7 +641,7 @@ bool Frame::FillRegionHistogramData(int region_id, CARTA::RegionHistogramData* h
                     } else {
                         std::unique_lock<std::mutex> guard(_image_mutex);
                         casacore::SubImage<float> sub_image;
-                        GetRegionSubImage(region_id, sub_image, curr_stokes, config_channel);
+                        GetRegionSubImage(region_id, sub_image, curr_stokes, {config_channel, config_channel});
```

Comment:
Should we not use the constructor with a single from/to value to avoid this repetition? We frequently use this kind of range.